src/features/security/_mixins.scss

```diff
@@ -4,13 +4,12 @@
     display: inline-flex;
     align-items: center;
     margin: 0;
-    padding: 0 5px;
-    border: 1px solid;
-    border-radius: $bdl-border-radius-size;
+    padding: 2px 8px;
+    border-radius: $bdl-border-radius-large-size;
 }
 
 @mixin bdl-SecurityBadge-name {
-    margin-left: 5px;
+    margin-left: 4px;
```

Comment:
we now have `$bdl-grid-unit` (4px) for margins/paddings. let's use where possible